warehouse/packaging/views.py

```diff
@@ -53,6 +57,53 @@ def project_detail(project, request):
     return release_detail(release, request)
 
 
+@view_config(
+    route_name="packaging.project_latest",
+    context=Project,
+)
+def project_latest(project, request):
+    return HTTPTemporaryRedirect(
+        request.route_path(
+            "packaging.release",
+            name=project.name,
+            version=project.latest_version.version,
```

Comment:
What if `project.latest_version` is None ?

Reply:
Fixed in dc838308